### ISSUE TITLE
Polkadot 0.9.43 cli breaking changes

### ISSUE THREAD

**Issue by kziemianek** [litentry/litentry-parachain#1774]
There were breaking changes in CLI parameters. We need to adjust our scripts to make them work again. For example our generated relaychain docker images stopped working with following error:

```
error: unexpected argument '--ws-external' found

  tip: a similar argument exists: '--rpc-external'
```

List of all cli changes:
https://github.com/paritytech/polkadot/releases/tag/v0.9.43

**Comment by kziemianek** [comment]
Some of the values are hardcoded in `open-web3-stack/parachain-launch`
https://github.com/open-web3-stack/parachain-launch/blob/01fbf1c4865ed1ce3b9844bc210e6fd81d06038f/src/index.ts#L488

Possible solution is to change `parity/polkadot` docker image tag from `latest` to last supported version (`v0.9.42`)

**Comment by Kailai-Wang** [comment]
We need to wait for a fix in parachain-launch (or we submit a PR ourselves)

**Comment by BillyWooo** [comment]
Hi @kziemianek , I changed the labels. Since our code base is still on 0.9.39, and [0.9.42](https://github.com/integritee-network/worker/pull/1337) is our next target. This breaking change won't affect us in a short time. And I don't think this is a bug for us.

**Comment by kziemianek** [comment]
The `parachain-launch` was recently updated to be compatible with parity/polkadot:v1.1.0

https://github.com/open-web3-stack/parachain-launch/commit/e15512533e9338c68c57909a04bf4c27f55e0e44

**Comment by kziemianek** [comment]
It looks like there is nothing more to be done in case of this ticket.

* we are fixed on parity/polkadot:v0.9.42 docker images - which is desired because we are stuck on that version
* once we update polkadot dep, we will try to use latest `parachain-launch`

Closing for now.
